lib/kitchen/provisioner/policyfile_zero.rb

```diff
@@ -96,9 +96,10 @@ def create_sandbox
       # (see Base#run_command)
       def run_command
         level = config[:log_level] == :info ? :auto : config[:log_level]
-        chef_client_bin = sudo(config[:chef_client_path])
+        
+        cmd = "#{sudo(config[:chef_client_path])} --local-mode".
+          tap { |str| str.insert(0, "& ") if powershell_shell? }
```

Comment:
Yeah this was a cut/paste job from the builtin provisioners in Test-Kitchen. Even better would be a helper method in `chef_base`or even `base` that all provisioners consumed. There is no spec coverage for this provisioner here however its never too late to start. Thats just gonna make this more than a trivial fix and I'll create a card for that. Basically any attempt to use this provisioner on a windows instance should blow up.